lib/omnibus/packager.rb

```diff
@@ -56,7 +57,13 @@ module Packager
     #
     def for_current_system
       family = Ohai['platform_family']
+      version = Ohai['platform_version']
 
+      if family == 'solaris2' && version >= '5.11'
```

Comment:
These comparisons should be done with a constraint:

```ruby
if family == 'solaris2' && version(version).satisfies?('>= 5.11')
```

Make sure you mixin the `Omnibus::Sugarable` module at the top:

```ruby
include Logging
include Sugarable
```